### ISSUE TITLE
 include executed/unexecuted state in cell descriptions

### ISSUE THREAD

**Issue by tonyfast** [deathbeds/nbconvert-a11y#50]
visual users can tell whether a cell is executed or unexecuted, screen reader users should be able too also.
